DataBiosphere/Saturn-UI
# [PF-692] Phase 2 regionality release

Follow up from #2660 . Phase 2 allows Cloud Environments to be created outside of us-central1. This becomes available if your workspace bucket is outside of the US.

To test:
1) Go to a workspace with a non-US bucket. If you need to create one, modify the availableBucketRegions function in region-common.js.
2) Open the Cloud Environment modal and click `CUSTOMIZE`. The location dropdown should be enabled and should have two choices - the bucket region and the US.
3) If you create the environment in the non-US region, you should see this warning:
![image](https://user-images.githubusercontent.com/6879774/141703562-b177e6de-5c01-47c4-aca0-96cc191f6296.png)

4) Create the environment!

This PR also updates the pricing estimates to be regional. Previously, price estimate data was looked up manually in the documentation and entered into machines.js. I have written a script that looks up the pricing data via Google's API in https://gist.github.com/wnojopra/be5be3ab7e6a09a351b89085992a940d (TODO: re-write this from Python to Node). You can copy-paste the output of that script into machines.js for updated pricing data. Follow the instructions in the script to run.
